wordpress-mobile/WordPress-iOS
# Jetpack Focus: Update promo screenshots lane

Part of #18960
Ref: pcdRpT-DL-p2

## Description
- Updated Jetpack promo screenshots to match the new design.
- Changed the iPad Pro 3rd generation to 5th generation as 3rd gen isn't available in the latest iOS.
- Turned off concurrent simulators as this was causing the screenshots lane to fail.
- Updated Jetpack promo strings, but they are **yet to be translated**.

## Screenshots

### iPhone XS Max

|Banner|1|2|3|4|5|
| - | - | - | - | - | - |
|![iPhone XS Max-0-Splash](https://user-images.githubusercontent.com/25306722/181671825-736e527d-dd4c-4d1d-b5ee-514e8c1107b4.png)|![iPhone XS Max-1-SiteCreation](https://user-images.githubusercontent.com/25306722/181671841-c964a602-b5a4-4056-aa04-65bb1e9659bd.png)|![iPhone XS Max-2-CreateNew](https://user-images.githubusercontent.com/25306722/181671855-113377fe-3219-46a4-8121-ab8927cda1af.png)|![iPhone XS Max-3-PageBuilder](https://user-images.githubusercontent.com/25306722/181671870-ced28fc5-a5f5-49db-8fe1-c04b218417f2.png)|![iPhone XS Max-4-Stats](https://user-images.githubusercontent.com/25306722/181671898-4be65f72-0a96-4f06-9bb9-13714851845a.png)|![iPhone XS Max-5-Notifications](https://user-images.githubusercontent.com/25306722/181671910-0ca7dbde-3f3e-4ad0-ad70-91a88660678f.png)|

### iPhone 8 Plus

|Banner|1|2|3|4|5|
| - | - | - | - | - | - |
|![iPhone 8 Plus-0-Splash](https://user-images.githubusercontent.com/25306722/181672126-6990ffd8-b313-476f-b9de-cdd9d557d326.png)|![iPhone 8 Plus-1-SiteCreation](https://user-images.githubusercontent.com/25306722/181672141-6dd21f69-d293-477d-b680-455e96011b98.png)|![iPhone 8 Plus-2-CreateNew](https://user-images.githubusercontent.com/25306722/181672156-79a5b6bc-3744-45b7-8aa8-cb7a1fb83f9c.png)|![iPhone 8 Plus-3-PageBuilder](https://user-images.githubusercontent.com/25306722/181672167-b8b95810-bb3f-4928-bf69-0e6e75d64646.png)|![iPhone 8 Plus-4-Stats](https://user-images.githubusercontent.com/25306722/181672183-ac4717d5-fc87-4c6d-ba97-ceb2ea8c8a65.png)|![iPhone 8 Plus-5-Notifications](https://user-images.githubusercontent.com/25306722/181672196-26063c2a-2c49-4d4a-95b8-e1b744b6abf1.png)|

### iPad Pro 2nd gen

|Banner|1|2|3|4|5|
| - | - | - | - | - | - |
|![iPad Pro (12 9-inch) (2nd generation)-0-Splash](https://user-images.githubusercontent.com/25306722/181673645-2837d1ea-d4eb-4748-84bf-0e33d1e7f454.png)|![iPad Pro (12 9-inch) (2nd generation)-1-SiteCreation](https://user-images.githubusercontent.com/25306722/181673683-9c2834c8-0109-4631-92e4-23292a9e6a4d.png)|![iPad Pro (12 9-inch) (2nd generation)-2-CreateNew](https://user-images.githubusercontent.com/25306722/181673712-3d1cb4b7-dc4a-442b-b651-8d8cf404482d.png)|![iPad Pro (12 9-inch) (2nd generation)-3-PageBuilder](https://user-images.githubusercontent.com/25306722/181673741-3894e87f-6b48-424d-939d-5147a4755330.png)|![iPad Pro (12 9-inch) (2nd generation)-4-Stats](https://user-images.githubusercontent.com/25306722/181673770-0296a1c4-e686-49b4-a980-ec806282962c.png)|![iPad Pro (12 9-inch) (2nd generation)-5-Notifications](https://user-images.githubusercontent.com/25306722/182274404-7ff44e72-a216-42df-9439-f79e52faee59.png)|

### iPad Pro 5th gen

|Banner|1|2|3|4|5|
| - | - | - | - | - | - |
|![iPad Pro (12 9-inch) (5th generation)-0-Splash](https://user-images.githubusercontent.com/25306722/186538086-9437ed71-fae8-4951-8a37-c4c1f64346d3.png)|![iPad Pro (12 9-inch) (5th generation)-1-SiteCreation](https://user-images.githubusercontent.com/25306722/186538096-37eeb89e-4a27-43ae-b54a-86e84ddff388.png)|![iPad Pro (12 9-inch) (5th generation)-2-CreateNew](https://user-images.githubusercontent.com/25306722/186538105-5ff3fc4f-c080-4c84-a27a-d12d9dcc4b9c.png)|![iPad Pro (12 9-inch) (5th generation)-3-PageBuilder](https://user-images.githubusercontent.com/25306722/186538113-d9a4135f-d5e8-4252-9815-2d61218c94e8.png)|![iPad Pro (12 9-inch) (5th generation)-4-Stats](https://user-images.githubusercontent.com/25306722/186538124-87c558ed-a982-4e66-bb1b-cff2af8f080c.png)|![iPad Pro (12 9-inch) (5th generation)-5-Notifications](https://user-images.githubusercontent.com/25306722/186538130-22a8a0cb-6a83-4618-8699-0c983618def3.png)|


## Testing Instructions

1. Make sure to have Inter font installed on your machine. You can get it from [here](https://rsms.me/inter/download/)
2. Run `rake mocks`
3. Run `bundle exec fastlane jetpack_screenshots language:en-US skip_clean:true`
4. Run `bundle exec fastlane create_jetpack_promo_screenshots force:true`
5. Check `fastlane/jetpack_promo_screenshots` for the final images

## Regression Notes
1. Potential unintended areas of impact
N/A

2. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

5. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
